src/test/java/org/mongoflink/source/MongoBatchSourceTest.java

```diff
@@ -64,13 +66,13 @@ public void testRead() throws Exception {
         ListSink<String> sink = new ListSink<>();
         env.fromSource(mongoSource, WatermarkStrategy.noWatermarks(), "mongo_batch_source")
                 .returns(String.class)
-                .addSink(sink);
```

Comment:
I think we should keep the original sink to verify the document number.

Reply:
yes. this commit is my mistake